client/templates/hangout/hangout-item.js

```diff
@@ -47,17 +47,18 @@ Template.hangoutItem.helpers({
   },
   getDate: function(hangout) {
     var tz = TimezonePicker.detectedZone();
+    var startEndDiffDays = (hangout.end - hangout.start)/(1000*60*60*24)
+    var startTime = moment(hangout.start).tz(tz).format('ddd MMMM Do YYYY, h:mm a z')
+    // If endtime is in 24 hours (95% of 24 hours), then show "To Be Announced". Else show the end time
+    var endTime = startEndDiffDays < 0.95 ? moment(hangout.end).tz(tz).format('MMMM Do h:mm a z'): 'To Be Announced '
+    var joinedUsers = hangout.users.length + ' joined'
```

Comment:
`getDate` , creates a string literal similar to this`Tue October 4th 2016, 3:18 am IST - October 4th 6:18 am IST | 3 joined ` which is being used by template. While I was working on #225 I need to use `getDate` for `/hangouts` view, and I realise that we are using it at multiple places, so I had done little bit of refactoring and converted it into `registerHelper` to reduce code repetition.